MakieCore/src/basic_plots.jl

```diff
@@ -383,7 +383,7 @@ Plots a marker for each element in `(x, y, z)`, `(x, y)`, or `positions`.
   Otherwise, one can set one color per point by passing a `Vector{<:Colorant}`, or one colorant for the whole scatterplot. If color is a vector of numbers, the colormap args are used to map the numbers to colors.
 - `cycle::Vector{Symbol} = [:color]` sets which attributes to cycle when creating multiple plots.
 - `marker::Union{Symbol, Char, Matrix{<:Colorant}, BezierPath, Polygon}` sets the scatter marker.
-- `markersize::Union{<:Real, Vec2f} = 9` sets the size of the marker.
+- `markersize::Union{<:Real, Vec2f} = 9` sets the size of the marker (twice the `markersize` is approximately four times as large).
```

Comment:
Perhaps something like:
```julia
- `markersize::Union{<:Real, Vec2f} = 9` sets the size of the marker (think radius, not area).

Reply:
I think it's better to keep the docstring somewhat vague and simple so people don't get overwhelmed by detail. `markersize` scales markers in x and y direction, either equally by a `Real` or independently by the components of a `Vec2f`. The units depend on `markerspace`.

The details about the final size of the marker can then go into the online documentation for people who actually care about it:
- for characters `markersize` is effectively the same as `fontsize`, i.e. `marker = m, markersize = 10, markerspace = :pixel` results in an `m` with a fontsize of 10px
- for `Circle` and `Rect` are unit-sized, i.e. `marker = Circle, markersize = 10, markerspace = :pixel` draws a circle with diameter = 10px.
- BezierPaths have their own coordinate system which gets scaled by markersize. E.g. if a Bezierpath includes points between 0..10 x 0..10, then `markersize = 10, markerspace = :pixel` will scale it to size of 0..100 by 0..100
- Symbols like `:circle` are BezierPaths with coordinate systems adjusted to work well with characters markers

> it scales width and height of the marker up for values greater than one, or down for values between zero and one
Also this made me think you were talking about scaling markers like how you'd resize an image (png, bmp, etc) at first. So just to clarify, in WGLMakie and GLMakie most markers do have a rasterized base size (32 and 64 respectively) but the resizing is much more accurate than a normal image would be. Circle and Rect markers are the exceptions here - they are effectively vector graphics. But this is all more of an implementation detail and not something we should have in the docs imo.

Reply:
Hm I was actually thinking that each attribute docstring _should_ contain all the information to really understand it (and the docs would just repeat it). Of course that is somewhat at odds with a manageable-length docstring for the whole plot type. That's why the descriptions of Block attributes have to be requested separately with `Axis.xlabel` etc. which doesn't work for plot types due to dispatch reasons.

Reply:
My vote is for concise completeness, with detail put into an extended help section. Is that an agreeable approach?